data/errors.json

```diff
@@ -0,0 +1,107 @@
+[
+    {
```

Comment:
I really like this database. Fun with errors. Did not know we hade this as an option in the example json files. Or maybe you found it somewhere else. Cool anyway.

Reply:
I asked ChatGpt to write such json format for me :)

Reply:
and then I just made a new file json and copied the text.